### ISSUE TITLE
Admin API: Too many open files, when installing hREA full suite

### ISSUE THREAD

**Issue by Connoropolous** [holochain/launcher-tauri#65]
**Describe the bug**
Application UI unresponsive. It cannot connect to the AppWebsocket. 

Note: hREA full suite is very large, about 75mb, and consists of many many zomes.

**To Reproduce**
Try installing this webhapp release: https://github.com/Connoropolous/hrea-dashboard/releases/tag/test-3
Note the 'Too many files open' error in the logs. 

**Expected behavior**
The app to install normally, and run normally

**Logs**
Please attach here the `launcher.log` file. To find it, go to `Holochain Admin` window of the Holochain Launcher, select the  `Settings` menu, and select `Open Logs`. Attach that file to this issue.

> In Ubuntu, there is a weird error that prevents the upload of a file from inside a hidden folder. To work around it, you can copy the `launcher.log` file to another non-hidden location, and upload it from there.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS v12.4
 - Holochain Launcher Version: v0.5.1

**Additional context**
[launcher.log](https://github.com/holochain/launcher/files/9543135/launcher.log)

**Comment by matthme** [comment]
I couldn't reproduce the error, everything seemed to work fine for me. Console log of the hREA Dashboard says connection to Holochain OK and the dashboard is perfectly responsive. I was able to register and create a resource. I'll have a look at your logs.

**Comment by Connoropolous** [comment]
Michael says there's a thing called ulimit which is set for each system and mine might be set low for some reason

**Comment by Connoropolous** [comment]
Yeah, I'm noticing it's working fine on another computer as well. An intel macos

**Comment by matthme** [comment]
I also saw this ulimit mentioned somewhere when searching for reasons for this error. It seems to me though that the Admin Websocket should support some mechanism to deal with this a scenario, I assume you won't be the only one.
What I'm not 100% sure is whether this warning was actually the cause for your problem since according to your log files you have this warning popping all the time. Did you have similar issues earlier?

**Comment by Connoropolous** [comment]
This solved the issue for me, but again looks like something not anybody could/would do:
https://mampsupportforum.com/viewtopic.php?t=155
